spec/integrations/rake_spec.rb

```diff
@@ -64,6 +65,8 @@
       result = request()
       expect(result["events"][0]["metaData"]["rake_task"]).not_to be_nil
       expect(result["events"][0]["metaData"]["rake_task"]["name"]).to eq("test:crash")
+      expect(result["events"][0]["app"]["type"]).to eq("rake")
+      expect(result["events"][0]["device"]["runtimeVersions"]["rake"]).to match(/\d+\.\d+\.\d+/)
```

Comment:
Anchor the regular expression (use `\A` in the beginning, although using `\z` at the end might be too brittle in cases where the version string is a prerelease or has 4 components).